packages/atlas/src/views/viewer/HomeView.tsx

```diff
@@ -1,92 +1,36 @@
 import styled from '@emotion/styled'
 import { FC } from 'react'
 
-import { useVideoHeroData } from '@/api/hooks/videoHero'
-import { useBasicVideosConnection } from '@/api/hooks/videosConnection'
 import { VideoOrderByInput } from '@/api/queries/__generated__/baseTypes.generated'
-import { GetMostViewedVideosConnectionDocument } from '@/api/queries/__generated__/videos.generated'
+import { GetBasicVideosConnectionDocument } from '@/api/queries/__generated__/videos.generated'
 import { InfiniteVideoGrid } from '@/components/InfiniteGrids'
-import { ViewErrorFallback } from '@/components/ViewErrorFallback'
-import { DiscoverChannels } from '@/components/_content/DiscoverChannels'
-import { NewNftSales } from '@/components/_content/NewNftSales'
-import { OfficialJoystreamUpdate } from '@/components/_content/OfficialJoystreamUpdate'
-import { TopTenVideos } from '@/components/_content/TopTenVideos'
 import { VideoContentTemplate } from '@/components/_templates/VideoContentTemplate'
-import { VideoHero } from '@/components/_video/VideoHero'
-import { atlasConfig } from '@/config'
-import { publicVideoFilter } from '@/config/contentFilter'
 import { useHeadTags } from '@/hooks/useHeadTags'
-import { usePersonalDataStore } from '@/providers/personalData'
 import { sizes, transitions } from '@/styles'
-import { SentryLogger } from '@/utils/logs'
 
 export const HomeView: FC = () => {
-  const followedChannels = usePersonalDataStore((state) => state.followedChannels)
-
-  const channelIdIn = followedChannels.map((channel) => channel.id)
-  const anyFollowedChannels = channelIdIn.length > 0
-
-  const { videoHero, loading } = useVideoHeroData()
-
-  const {
-    videosConnection,
-    loading: followedLoading,
-    error: followedError,
-  } = useBasicVideosConnection(
-    {
-      where: {
-        ...publicVideoFilter,
-        channel: {
-          id_in: channelIdIn,
-        },
-      },
-    },
-    { skip: !anyFollowedChannels, onError: (error) => SentryLogger.error('Failed to fetch videos', 'HomeView', error) }
-  )
-
   const headTags = useHeadTags()
 
-  const followedChannelsVideosCount = videosConnection?.totalCount
-
-  if (followedError) {
-    return <ViewErrorFallback />
-  }
-
   return (
     <VideoContentTemplate>
       {headTags}
-      <VideoHero videoHeroData={videoHero} withMuteButton loading={loading} />
       <Container className={transitions.names.slide}>
-        {!followedLoading && followedChannelsVideosCount ? (
-          <InfiniteVideoGrid
-            title="Followed channels"
-            videoWhereInput={{ channel: { id_in: channelIdIn } }}
-            ready={!followedLoading}
-            onDemand
-            titleLoader
-          />
-        ) : null}
         <InfiniteVideoGrid
```

Comment:
I would use the `Section` component here since we are using it already in other places. Currently, we have two components that do practically the same thing, I would remove InfiniteVideoGrid and all related logic to it entirely(I think it's only used in HomeView anyway)